cas-server-support-jpa-service-registry/src/main/java/org/apereo/cas/config/JpaServiceRegistryInitializer.java

```diff
@@ -0,0 +1,54 @@
+package org.apereo.cas.config;
+
+import org.apereo.cas.services.ServiceRegistryDao;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+
+/**
+ * Initializes Jpa service registry database with available JSON service definitions if necessary (based on configuration flag).
+ *
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Empty line should be followed by <p> tag on the next line.](https://www.codacy.com/app/mmoayyed/cas/file/2569584858/issues/source?bid=3256115&fileBranchId=3297474#l16)